src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/HashHelpers.cs

```diff
@@ -47,9 +49,8 @@ public static int GetPrime(int min)
             if (min < 0)
                 throw new ArgumentException(SR.Arg_HTCapacityOverflow);
 
-            for (int i = 0; i < primes.Length; i++)
+            foreach (int prime in Primes)
```

Comment:
Just noting that while C# lowers this down to a `for` loop, I believe its a slightly different pattern and isn't always optimized quite as well.

It's something like an instruction or two difference, and probably not meaningfully significant, but might be something to double check.

Reply:
It looks identical: [SharpLab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYRoG8GuGbunHi5JAwCWAOwwMA4jAwAFKCPwxyACnGT84gJS9u7PXy4AlGNgAmAeTEAbAJ4BlAA7YxAHg0A+Bk8XLcDAC8DApKMLgA3IZGAGbQDOoSokEMAAwRyW4+fuEAdAAyMGIA5hgAFhkiANRVutRG+tENXBrZYSm+YbgA2iIAulH1zXwiMQmdygyewVpidcPDxADsbcqDwwC+NE3cywxw5OvcW9TR9ExCokky8jmkiZo60QZDDXGw2GBlCa0TMFchHK4ebNF4LbijcY5KYzJ6vcG7FZ/I4NE47Lh7A4ohho+HnQTCUwWaz2ZyuDwSbyhfxBbxiGAAdyuGG6fWe6IYdDQDCW3PI5D5vIYpC5woAnNy6EKUEKAKwSnkChgADgV5FSQvIdCVQlF5DFQpFCtIYtFdFlopQ2u5soo3KQ1p5SCVYv1aA5/NSeropE18s1pqVIrootIytSSpDSyVKFSKAVKGVzptyroCqWEdFypQboY6vVeooVr5KHlAo9S1luYoqRF3NItpNktrpslDot3KtSF9NpFyYY3eVVe5latLtStr1qSTjq1XodfNlqCl7vhfHIyqWlaDDboGvrGalkdIKFPCroqvN0qQSxFcqQqSQqvtt+VPZVGtjorFT9VmqEpBPpqaa2guHo3mOxqpKkrquvWOYFqGl4nueMobmKCZCIO0oYWKsryjauFoSO+qmkgYrRBsgwbEAA===)